lmfit/parameter.py

```diff
@@ -75,12 +75,19 @@ def __init__(self, asteval=None, usersyms=None, *args, **kwds):
         for key, val in _syms.items():
             self._asteval.symtable[key] = val
 
-        self.update(*args, **kwds)
-
     def copy(self):
         """Parameters.copy() should always be a deepcopy."""
         return self.__deepcopy__(None)
 
+    def update(self, other):
+        """update values with another Paramters object"""
```

Comment:
how about ```Update values and symbols with another Parameters object.``` - correcting a typo, adding "symbols" and using uppercase/punctuation.